src/services/pocket-relayer.ts

```diff
@@ -646,3 +674,14 @@ interface BlockchainDetails {
   blockchainIDCheck: string
   blockchainID: string
 }
+
+export interface SendRelayOptions {
```

Comment:
Any reason to prefer interface over types? This is fine by me as it's a matter of preference I believe, just wanna make sure we stick to a standard.

Reply:
yeah, that's definitely a type, good catch! the standard here should be that unless it can't be done by a type (for example inheritance) then use a type